### ISSUE TITLE
Consider adding Testing library with Jest

### ISSUE THREAD

**Issue by MarcoEscaleira** [jakoblind/webpack-autoconf#147]
Raising a possible issue where we could add Testing library integration out of the box.

**Comment by jakoblind** [comment]
We already have support for this. 
![image](https://user-images.githubusercontent.com/399339/133969773-6e65391d-345c-4d88-b599-b0c3b531c210.png)
Any particular test library you are missing?

**Comment by MarcoEscaleira** [comment]
Yes in terms of frameworks we have those, my suggestion here is towards having the options to include Testing Library as well as other utilities that help testing code within these frameworks
